packages/eslint-plugin/src/rules/best-practices/no-discouraged-words.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Rule } from 'eslint';
+import { TSESTree } from '@typescript-eslint/types';
+import { createDocsUrl } from '../../util/helpers';
+
+/**
+ * List of discouraged words in the form of RegEx (regular expressions).
+ */
+const discouragedWords = ['blacklist', 'whitelist'];
```

Comment:
Ideally we would store this discouragedWords list somewhere centrally that could be imported/inherited from, without having to reproduce the list verbatim in each repo for each linter.

We'll be breaking new ground, so input from other folks on where/how best to do that would be helpful and appreciated to align each of our implementations.